index.html

```diff
@@ -527,6 +527,76 @@ <h3>GpgVerificationKey2020</h3>
   </pre>
   </section>
 
+  <section>
+    <h2>Workday Signature Suite</h2>
+
+    <h3>WorkEd25519Signature2020</h3>
+
+    <table class="simple">
+      <thead>
+        <tr>
+          <th colspan=2>Summary</th>
+        </tr>
+      </thead>
+
+      <tbody>
+        <tr>
+          <td>Identifiers</td>
+          <td>WorkEd25519Signature2020</td>
+        </tr>
+        <tr>
+          <td>Status</td>
+          <td>PROVISIONAL</td>
+        </tr>
+        <tr>
+          <td>Authors</td>
+          <td>Workday</td>
+        </tr>
+        <tr>
+          <td>Specification</td>
+          <td><a href="https://github.com/w3c-ccg/ld-cryptosuite-registry/blob/gh-pages/doc/workday/signature-suite.md">Workday Signature Suite 2020</a></td>
+        </tr>
+      </tbody>
+    </table>
+
+    <h3>WorkEd25519VerificationKey2020</h3>
+
+    <table class="simple">
+      <thead>
+        <tr>
+          <th colspan=2>Summary</th>
+        </tr>
+      </thead>
+
+      <tbody>
+        <tr>
+          <td>Identifiers</td>
+          <td>WorkEd25519VerificationKey2020</td>
+        </tr>
+        <tr>
+          <td>Status</td>
+          <td>PROVISIONAL</td>
+        </tr>
+        <tr>
+          <td>Authors</td>
+          <td>Workday</td>
+        </tr>
+        <tr>
+          <td>Specification</td>
+          <td><a href="https://github.com/w3c-ccg/ld-cryptosuite-registry/blob/gh-pages/doc/workday/signature-suite.md">Workday Signature Suite 2020</a></td>
+        </tr>
+      </tbody>
+    </table>
+
+    <pre class="example nohighlight" title="Example of WorkdayEd25519Signature2020">{
+   "created": "2020-02-21T22:37:48Z",
+    "creator": "did:work:6sYe1y3zXhmyrBkgHgAgaq#key-1",
+    "nonce": "524a497a-871c-4b89-a325-205f78fc5fcd",
+    "signatureValue": "2NuYkfkFy8o4R8iRCaLzTYc1Uss8s22rvC7jiY4BA49sFmWbwbEkQ6BHQPULEp7mPfQQvJdKSjKRLJF8sp62GKvo",
```

Comment:
this does not look like a detached jws.... this is fine if thats your intention.